nbgrader/tests/apps/test_config.py

```diff
@@ -0,0 +1,15 @@
+from traitlets.config import Config
+from ...coursedir import CourseDirectory
+from .base import BaseTestApp
+from .conftest import notwindows
+
+class TestCourseDirectory(BaseTestApp):
+
+    @notwindows
+    def test_submit_rectoryget_source_assignments(self):
```

Comment:
Is the title of this method a typo?

Reply:
Oops, yes definitely. No idea how it became this :-) Fixed and pushed!